velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -404,10 +420,28 @@ struct GetTimestampFunction {
                                    : sessionTimeZone_;
   }
 
+  bool initializeFormatter(const std::string_view& format) {
```

Comment:
I find we could combine 'setFormatter' and 'initializeFormatter' into one and put it in the anonymous namespace. Perhaps return the formatter or nullptr for an invalid case.

Reply:
Thanks. Refactored this part.